kernel/packages/shared/apis/QuestsController.ts

```diff
@@ -0,0 +1,33 @@
+import { ExposableAPI } from './ExposableAPI'
+import { exposeMethod, registerAPI } from 'decentraland-rpc/lib/host'
+import { ClientResponse, PlayerQuestDetails, ProgressData } from 'dcl-quests-client'
+import { questsRequest } from '../quests/client'
+
+export type QuestsResponse<T> = Omit<ClientResponse<T>, 'status'>
+
+@registerAPI('QuestsController')
```

Comment:
We used to expose publicly visible modules with an experimental prefix to warn everyone and cover ourselves to make improvements